internal/cli/bridge.go

```diff
@@ -28,7 +32,7 @@ func BridgeCMD(toolName string) *cli.Command {
 
 		For example:
 
-		$ sudo %s bridge --network-token <TOKEN>
+		$ sudo %s bridge --token <TOKEN>
```

Comment:
Removed network-token because, when erroring, it shows the edgevpn environment variable names, not the ones previously defined here

```
kairosctl bridge
{"level":"FATAL","time":"2024-04-24T15:14:34.867+0200","caller":"cli/bridge.go:174","message":" EDGEVPNCONFIG or EDGEVPNTOKEN not supplied. At least a config file is required"}
```